linux_os/guide/system/bootloader-grub2/file_permissions_grub2_cfg/oval/shared.xml

```diff
@@ -0,0 +1,40 @@
+<def-group>
+  <definition class="compliance" id="file_permissions_grub2_cfg" version="1">
+    <metadata>
+      <title>Verify /boot/grub2/grub.cfg Mode Permissions</title>
+      <affected family="unix">
+        <platform>multi_platform_all</platform>
+      </affected>
+      <description>This test makes sure that /boot/grub2/grub.cfg has mode 0600.
+      If the target file or directory has an extended ACL, then it will fail the mode check.
```

Comment:
Should be `File permissions for /boot/grub2/grub.cfg should be set to 0600 (or stronger).`